install-deps.sh

```diff
@@ -517,6 +517,11 @@ else
                 $SUDO rpm --import /etc/pki/rpm-gpg/RPM-GPG-KEY-EPEL-$MAJOR_VERSION
                 $SUDO rm -f /etc/yum.repos.d/dl.fedoraproject.org*
                 if test $ID = centos -a $MAJOR_VERSION = 8 ; then
+                    # for grpc-devel
+                    # See https://copr.fedorainfracloud.org/coprs/ceph/grpc/
+                    $SUDO yum -y install https://dl.fedoraproject.org/pub/epel/epel-release-latest-8.noarch.rpm
```

Comment:
Isn't the script already enabling epel 4-5ish lines above? Is this line necessary?

Reply:
On top of that, this line is the only one that invokes yum, so if you keep it, can we consistently use 'dnf'? Thanks.

Reply:
Thank you John 🖖 Appreciate your review. See [diff](https://github.com/ceph/ceph/pull/54671/commits/93f38b00ac1f49b5879967797681b3ea219f23bb).